model-optimizer/mo/utils/cli_parser.py

```diff
@@ -88,12 +87,12 @@ def __call__(self, parser, namespace, values, option_string=None):
 
 def readable_file(path: str):
     """
-    Check that specified path is a readable file.
+    Check that specified path is a readable file or directory.
     :param path: path to check
-    :return: path if the file is readable
+    :return: path if the file/directory is readable
     """
-    if not os.path.isfile(path):
-        raise Error('The "{}" is not existing file'.format(path))
+    if not os.path.isfile(path) and not os.path.exists(path):
+        raise Error('The "{}" is not existing file or directory'.format(path))
```

Comment:
This is a wrong change. We have a dedicated check that the path is directory. See below.

Reply:
Fixed, had to introduce readable_file_or_dir method